source/dofs/dof_renumbering.cc

```diff
@@ -2248,6 +2254,146 @@ namespace DoFRenumbering
            ExcInternalError());
   }
 
+
+
+  template <int dim, int spacedim>
+  void
+  support_point_wise(DoFHandler<dim, spacedim> &dof_handler)
+  {
+    std::vector<types::global_dof_index> renumbering(
+      dof_handler.n_locally_owned_dofs(), numbers::invalid_dof_index);
+    compute_support_point_wise(renumbering, dof_handler);
+
+    // If there is only one component then there is nothing to do, so check
+    // first:
+    if (Utilities::MPI::max(renumbering.size(),
+                            dof_handler.get_communicator()) > 0)
```

Comment:
I think I don't understand the "only one component" here. `renumbering` has `n_locally_owned_dofs()` size, how does that relate to the number of components?

Reply:
Same as the other comment - this array will be resized to have a length of zero if there is nothing to be done for the component renumbering.